src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -44,7 +47,7 @@ public function normalize($object, $format = null, array $context = [])
         $violations = [];
         $messages = [];
         foreach ($object as $violation) {
-            $propertyPath = $violation->getPropertyPath();
+            $propertyPath = $this->nameConverter ? $this->nameConverter->normalize($violation->getPropertyPath()) : $violation->getPropertyPath();
```

Comment:
You should also pass the current context, to respect the AdvancedNameConverterInterface.

Reply:
Updated